client/src/client.cpp

```diff
@@ -88,38 +92,91 @@ void rtype::Client::drawParallax(sf::RenderWindow &window)
     window.draw(parallaxSprite3b);
 }
 
+ECS rtype::Client::initECS()
+{
+    ECS ecs;
+    ecs.registerComponent<Position>();
+    ecs.registerComponent<Health>();
+    ecs.registerComponent<Velocity>();
+    return ecs;
+}
+
 /**
  * @brief Run the client
  */
```

Comment:
Update Doxygen code comment with new params.

example:
* @param sender The network sender object responsible for sending messages.
 * @param receive The network receiver object responsible for receiving messages.
 * @param port The UDP port used for network communication.